src/main/java/uk/antiperson/stackmob/listeners/TagInteractListener.java

```diff
@@ -35,5 +35,7 @@ public void onTagInteract(PlayerInteractEntityEvent event) {
             return;
         }
         stackEntity.slice();
+        stackEntity.removeStackData();
+        stackEntity.getEntity().setCustomNameVisible(false);
```

Comment:
Perhaps this should be moved to the #removeStackData method?

Reply:
Good idea, done ^^